go/test/endtoend/cluster/cluster_process.go

```diff
@@ -173,6 +174,8 @@ func (cluster *LocalProcessCluster) StartTopo() (err error) {
 	if cluster.Cell == "" {
 		cluster.Cell = DefaultCell
 	}
+
+	topoFlavor = cluster.TopoFlavorString()
```

Comment:
This variable is hiding the global, which is confusing. You could as well call the func directly in line 181.

Reply:
We are updating the global. global is used multiple places and this is seems a viable option as it is a test framework